### PR TITLE
feat: connect your github sounds better

### PR DESCRIPTION
**Pull request by bdougie**
## Description

Had an instance where I got to watch someone try to log into our app from the landing page and many questions came up. It is not clear what "connecting with github" means in the context of our landing page. I am swapping the with to your in this PR.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Mobile & Desktop Screenshots/Recordings

before:

Screen Shot 2024-03-15 at 10 02 06 AM

after:

Screen Shot 2024-03-15 at 10 02 56 AM

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
There are extra changes in here. Looks like my prettier decided to work. I am going to roll with it, but we may want to consider aligning the lint rules with other projects. 

https://github.com/open-sauced/landing-page/blob/main/.prettierrc.js#L5

**Review by nickytonline — APPROVED** [review]
:shipit:

File: components/sections/navigation/DesktopNavigation.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { useRouter } from 'next/router'
 
 // Components
 import { Button } from '../../common'
-import { Menu } from "@headlessui/react";
+import { Menu } from '@headlessui/react'
```
Comment (jpmcb) [inline components/sections/navigation/DesktopNavigation.tsx:8]: Maybe not blocking, but this changes a number of `"` double quotes over to `'` single quotes: we can push this pr through but would probably be good to get the eslint/prettier checks in here along with a `package.json` script to easy execute linting

Comment (nickytonline) [reply components/sections/navigation/DesktopNavigation.tsx:8]: It looks like prettier was kicking in. Not sure why it's so different though.

Comment (jpmcb) [reply components/sections/navigation/DesktopNavigation.tsx:8]: We can move this forward: maybe we re-visit linting/formatting in the future 👀

**Pull request merged**